src/style.css

```diff
@@ -0,0 +1,68 @@
+/* stylelint-disable color-hex-length */
+@import url("https://fonts.cdnfonts.com/css/avenir");
```

Comment:
right now your demo link has too long response time according to uploading fonts from this link
you're already have fonts folder in the project, just check in video how to add fonts via @font in correct way, and use existing fonts for your elements